RecoBTag/DeepFlavour/python/DeepFlavourJetTags_cfi.py

```diff
@@ -0,0 +1,23 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
file name pfDeepFlavourJetTags_cfi.py

Reply:
will be removed, now using fillDescriptions

Reply:
using fillDescriptions since https://github.com/cms-sw/cmssw/pull/19893/commits/c2dc5c01427a4ea10ed8cba83f44382c986ebb46